src/components/GraphFilter/GraphFilter.tsx

```diff
@@ -82,19 +93,14 @@ export default class GraphFilter extends React.PureComponent<GraphFilterProps> {
             label={GraphFilter.GRAPH_TYPES[graphTypeKey]}
             options={GraphFilter.GRAPH_TYPES}
           />
-          <span style={{ marginLeft: '1.5em' }}>
-            <ToolbarDropdown
-              id={'graph_filter_interval_duration'}
+          <Toolbar.RightContent>
+            <GraphRefreshContainer
```

Comment:
I'm not sure to understand why these two components (duration and refresh options) are merged into one.
In this PR [ #662 ] I'm re-using the refresh component but I didn't plan to use the duration one.
Of course if there are good reasons to merge them, then np I'll refactor my code

Reply:
Good point @jotak. For flexibility it should probably be 2 separate components which can be grouped by another component for pages that want that

Reply:
In my point of view, duration and refresh are related, because changing either one triggers a refresh.

Hmm... If we want different components grouping the controls, it's either @jotak or me who need to do a refactor.

An alternative I was thinking is just to add flags to choose which controls to show. This will lead to minor changes from both @jotak and me.

You tell... It's fine for me both alternatives (flags vs more components).

Reply:
BTW, if you are re-using... does that mean that you want the overview refresh synced with the graph page?
Given that the refresh is being stored in redux, when you switch views, both will be synced whith the last value chosen in the last visited page.

Reply:
The whole idea of the component architectures is to break things down into their smallest atomic components. That way, the components are smaller, more reusable, isolated and easily unit tested.

Reply:
We probably need a talk & coordination about the UXD / layout changes. Changes I see here are contradictory with what's requested on the overview page task.

Reply:
Added DNM label, just to sync with all parties.

Reply:
So, after discussion I changed my PR to use a different Refresh component, so it won't conflict anymore.
The problem of having various "refresh" components isn't solved but there's already a task for dealing with it ( https://issues.jboss.org/browse/KIALI-1476 ), so it can be done later.